www/js/components/add-point-card.js

```diff
@@ -16,8 +16,8 @@ const defaultState = {
   name: '',
   description: '',      // description
   checkIn: false,       // mark if they were there or not
-  imgSrc: '',           // image url
-  imgBlob: '',          // image blob
+  coverUrl: '',           // image url
+  coverBlob: '',          // image blob
```

Comment:
Changed these variable names throughout to reflect they are blobs and url's for the card cover. We may eventually have more images.